openssl-sys/src/lib.rs

```diff
@@ -1098,14 +719,30 @@ extern "C" {
                         -> c_int;
     pub fn PKCS12_free(p12: *mut PKCS12);
 
-    pub fn sk_free(st: *mut _STACK);
-    pub fn sk_pop_free(st: *mut _STACK, free: Option<unsafe extern "C" fn (*mut c_void)>);
-    pub fn sk_pop(st: *mut _STACK) -> *mut c_char;
+    #[cfg_attr(any(ossl101, ossl102), link_name = "sk_free")]
```

Comment:
It doesn't look like the usage of these function's can't be unified due to the different representation of `_STACK`, so it doesn't seem like munging the name of the function on 1.0.x buys us anything other than confusing people searching for the 1.0.x name.